src/containers/ActiveZoneSelector/ActiveZoneSelector.js

```diff
@@ -57,7 +57,8 @@ class ActiveZoneSelector extends Component {
       top: '50%',
       transform: 'translateY(-50%)',
       width: '200px',
-      marginLeft: '170px'
+      marginLeft: '170px',
+      zIndex: '1'
```

Comment:
This only happens in multi zoned integrations but I added to general because why not. The solution is already a mess. Tested on WordPress and works fine